### ISSUE TITLE
Heatmap doesn't render negative values correctly

### ISSUE THREAD

**Issue by ryinyang** [apexcharts/apexcharts.js#2937]
### Description

The heat map doesn't render the colors correctly if there is a negative value. I've tried specifying `options.plotOptions.heatmap.min` and `max` but no success.

### Steps to Reproduce

1. Create heatmap containing a negative value

### Expected Behavior

Cell with lowest value should be white and linearly have more color saturation as value goes higher.

### Actual Behavior

All cells which I would expect to be white are light blue.

### Screenshots

![image](https://user-images.githubusercontent.com/16808920/151470609-e6d66d4b-6ae8-446f-b049-39d24e5f7f92.png)

### Reproduction Link

https://codesandbox.io/s/react-basic-example-forked-15vvp?file=/src/App.js

**Comment by junedchhipa** [comment]
Try the `shadeIntensity` property to control how much whiteness you need

```
  plotOptions: {
    heatmap: {
      shadeIntensity: 0.1,
    }
  }
```

**Comment by ryinyang** [comment]
I believe `shadeIntensity` still doesn't achieve the behavior I'm expecting. Correct me if I'm wrong, but I believe that the color of the cell should be on a linear scale from the min to the max. So if the range for HeatMapA is from 0 to 100, a cell with value 0 should be white and cell with value 10 should be very light blue. Likewise, if the range for HeatMapB is from -10 to 90, a value of -10 should be white and a cell with value 0 should have a very light blue.

If I set `shadeIntensity` to something like 0 on HeatMapB, a cell with value -10 correctly renders white, but a cell with value 0 is as rendering just as white as -10. I expect that the cell with value 0 should look light blue.

Using my above assumption for the expect behavior, I believe that if HeatMapA has cells of value 0, 10, 100, it should visually look exactly the same as HeatMapB with cells of value -10, 0, 90 (values are just shifted -10).

I've updated my codesandbox to illustrate that HeatMapA and HeatMapB don't look alike: https://codesandbox.io/s/react-basic-example-forked-15vvp?file=/src/App.js

![image](https://user-images.githubusercontent.com/16808920/151617228-fdde1130-d529-452b-a730-bf72d6adf170.png)
